### PR TITLE
Replaced WebL10n in Abacus Activity

### PR DESCRIPTION
**Pull request by Vishalk91-4**
Issue #1378 
Replaced WebL10n in Abacus Activity

@llaske could you please review it

### CODE REVIEW

**Comment by llaske** [comment]
Seems to be a good start but the tutorial no longer work:

image

**Comment by Vishalk91-4** [comment]
I'll look into this and change it

**Comment by ShrutiSharma77** [comment]
I apologize if my approach caused any confusion. I'll work on adjusting the code to align with both migration guides and find a solution that doesn't involve an assignment operator.

File: activities/Abacus.activity/lib/l10n.js
Comment (llaske) [inline activities/Abacus.activity/lib/l10n.js:1]: The code of this file has been updated [here](https://github.com/llaske/sugarizer/blob/dev/lib/l10n.js).
Could you update your PR with the recent version?

**Comment by llaske** [comment]
Fixed in https://github.com/llaske/sugarizer/pull/1445

**Pull request closed without merging**